x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/100_usage.yml

```diff
@@ -0,0 +1,40 @@
+setup:
```

Comment:
This test looks great! 

Some other test cases you could consider: 
- count = 0
- Add a new search application and verify the count increases / remove and verify it decreases